layouts/partials/cards/_footer.html

```diff
@@ -0,0 +1,11 @@
+{{ if isset . "readMoreLink" }}
+<div class="border-t border-card-dark pt-5 text-right">
+  <a href="{{ .readMoreLink }}" class="link">
+    {{ $readMoreText := "Read more" }}
+    {{ if isset . "readMoreText"}}
+      {{ $readMoreText = .readMoreText }}
+    {{ end }}
+    {{ $readMoreText }} »
```

Comment:
Ahh, I'm so sorry, but I missed this too. You can probably simplify this whole section to

```html
{{ default "Read more" .readMoreText }} »
```

On the one hand it's nitpicking, on the other hand, it's *80% line saving*, which makes me feel like [Skeletor when he saves money](https://www.youtube.com/watch?v=nqhLn76kCv0)